storage/rocksdb/ha_rocksdb.cc

```diff
@@ -1414,6 +1414,18 @@ static constexpr ulonglong RDB_DEFAULT_MAX_COMPACTION_HISTORY = 64;
 static constexpr ulong ROCKSDB_MAX_MRR_BATCH_SIZE = 1000;
 static constexpr uint ROCKSDB_MAX_BOTTOM_PRI_BACKGROUND_COMPACTIONS = 64;
 
+static constexpr char large_prefix_deprecated_msg[] =
+    "using rocksdb_large_prefix is deprecated and it will be removed in a "
+    "future release";
+
+static void rocksdb_large_prefix_update(THD *thd, SYS_VAR *, void *var_ptr,
+                                        const void *save) {
+  push_warning(thd, Sql_condition::SL_WARNING, HA_ERR_WRONG_COMMAND,
+               large_prefix_deprecated_msg);
+
+  *static_cast<bool*>(var_ptr) = *static_cast<const bool*>(save);
```

Comment:
could we add a check that if DDSE is rocksdb, user can't turn off rocksdb_large_prefix